tox.ini

```diff
@@ -14,6 +14,7 @@ basedeps = mock>=1.3.0
 deps = {[testenv]basedeps}
        django
        keyring
+       jsonpickle
```

Comment:
See an [example][1] of extras. You'd just give a list like

```python
DJANGO_EXTRAS = ('django', 'jsonpickle')

setup(
    ...
    extras_require={'django': DJANGO_EXTRAS},
    ...
)
```

and then people could install via

```
pip install oauth2client[django]
```

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/f02d7a28cc835790cf7991da7323416769463888/setup.py#L49